src/main/java/org/jitsi/videobridge/cc/vp8/VP8FrameProjection.java

```diff
@@ -425,11 +428,13 @@ private void rewriteRtpInternal(@NotNull Vp8Packet pkt)
         pkt.setSsrc(ssrc);
         pkt.setTimestamp(timestamp);
 
-        int sequenceNumberDelta = RTPUtils.getSequenceNumberDelta(
-            pkt.getSequenceNumber(), vp8Frame.getStartingSequenceNumber());
+        int sequenceNumberDelta
+                = RtpUtils.getSequenceNumberDelta(
+                    pkt.getSequenceNumber(),
+                    vp8Frame.getStartingSequenceNumber());
 
-        int sequenceNumber = RTPUtils.applySequenceNumberDelta(
-            startingSequenceNumber, sequenceNumberDelta);
+        int sequenceNumber
+            = (startingSequenceNumber + sequenceNumberDelta) & 0xFFFF;
```

Comment:
do we not have something like `applySequenceNumberDelta` anymore?

Reply:
I don't see it in `RtpUtils`. I guess most places use `RtpSequenceNumber`, and I don't think I can use it in java because it's an inline class. Can we leave this for the next next round of cleanup?

Reply:
Easy enough to add this to jitsi-rtp, I can do this.

Reply:
https://github.com/jitsi/jitsi-rtp/pull/44